client/my-sites/subscribers/components/subscribers-header/subscribers-header.tsx

```diff
@@ -31,41 +27,19 @@ type SubscribersHeaderProps = {
 	hideSubtitle?: boolean;
 };
 
-const HELP_CENTER_STORE = HelpCenter.register();
-
 export const SubscribersHeader = ( {
 	selectedSiteId,
 	disableCta,
 	hideSubtitle,
 }: SubscribersHeaderProps ): ReactElement => {
 	const localizeUrl = useLocalizeUrl();
-	const { setShowSupportDoc } = useDataStoreDispatch( HELP_CENTER_STORE );
 	const siteId = useSelector( getSelectedSiteId ) ?? null;
-	const isWPCOMSite = useSelector( ( state ) => getIsSiteWPCOM( state, siteId ) );
-
-	const openHelpCenter = () => {
-		setShowSupportDoc( localizeUrl( 'https://wordpress.com/support/paid-newsletters/' ) );
-	};
-
-	const paidNewsletterUrl = ! isWPCOMSite
-		? 'https://jetpack.com/support/newsletter/paid-newsletters/'
-		: 'https://wordpress.com/support/paid-newsletters/';
+	const subscribersLearnMoreUrl =
+		'https://wordpress.com/support/category/grow-your-audience/newsletter/';
 
 	const subtitleOptions = {
 		components: {
-			link: (
-				<a
-					href={ localizeUrl( paidNewsletterUrl ) }
-					target="blank"
-					onClick={ ( event ) => {
-						if ( ! isJetpackCloud() ) {
-							event.preventDefault();
-							openHelpCenter();
-						}
-					} }
-					rel="noreferrer"
-				/>
-			),
+			link: <a href={ localizeUrl( subscribersLearnMoreUrl ) } target="blank" rel="noreferrer" />,
```

Comment:
Just checking this change is intentional: we're now linking to the same support doc for both Calypso Blue and Jetpack Cloud, and no longer opening the help center.

Reply:
Thank you! Yes, that's the current behavior, to open the help center on Dotcom but a new tab for Jetpack. 

Atomic and Simple Classic sites can also open subscribers in Jetpack Cloud. But they should continue to link to the Dotcom docs in a new tab. That's what I'm seeing on the PR. 👍